BD-2484 Performance data by segment (#6269)
braze-inc/braze-docs: _docs/_user_guide/data_and_analytics/reporting/viewing_and_understanding_segment_data.md

{% endalert %}

![The Segment breakdown page contains a SQL editor, a side panel with tabs for Variables, Available Data Tables, Query History, and the AI Query Builder, and a results section.]({% image_buster /assets/img_archive/segment_breakdown.png %})

You can use [Query Builder]({{site.baseurl}}/user_guide/data_and_analytics/query_builder/) report templates to break down performance metrics for campaigns, Canvas, variants, and steps by segments. [Analytics tracking]({{site.baseurl}}/user_guide/data_and_analytics/tracking/segment_analytics_tracking#segment-analytics-tracking) must be turned on for the segments you want to access metrics for.

To run these reports, do the following:
1. In **Query Builder**, choose to create a new SQL report via a template. 
2. Filter templates for those where the metrics include “segment breakdowns”, which are:
- Email performance metrics by segment
- Purchases and revenue by segment

See below for details on each templated report.

{% tabs %}
{% tab Email engagement metrics by segment %}

### Viewing metrics for campaigns or Canvases {#campaign-canvas-email}

> To view email performance metrics broken down by segment at the campaign or Canvas level, use the [Variables](#variables) tab to specify the campaign or Canvas, as well as a time frame for pulling data.

Reviewer: ```suggestion
To view email performance metrics broken down by segment at the campaign or Canvas level, use the [Variables](#variables) tab to specify the campaign or Canvas and a time frame for pulling data.
```

The following email metrics are available in this report:
- Sends
- Deliveries
- Complaints
- Unique opens
- Unique machine opens
- Unique non-machine opens
- Unique clicks
- Unsubscribes
- Bounces
- Soft bounces
- Deferred

Metrics from non-email channels are not yet available.

### Viewing metrics for variants or steps

To view email performance broken down by segment at the campaign variant level, Canvas variant level, or Canvas step level, use the [Variables](#variables) tab to specify the following:
- Specific campaign or Canvas
- Variants 

> - (Optional) Canvas step

Reviewer: ```suggestion
- Canvas step (optional)
```
